cms/djangoapps/contentstore/rest_api/v2/views/home.py

```diff
@@ -90,9 +98,10 @@ def get(self, request: Request):
         """
 
         courses, in_process_course_actions = get_course_context_v2(request)
-        courses_context = {
-            "courses": courses,
-            "in_process_course_actions": in_process_course_actions,
-        }
-        serializer = CourseHomeTabSerializerV2(courses_context)
-        return Response(serializer.data)
+        paginator = PageNumberPagination()
+        courses_page = paginator.paginate_queryset(courses, self.request, view=self)
+        serializer = CourseHomeTabSerializerV2({
+            'courses': courses_page,
+            'in_process_course_actions': in_process_course_actions,
```

Comment:
Note to reviewer: I'm unsure how to handle this case. In this view, not only courses are returned, but also returns `in_process_course_actions` which is a list of CourseRerunState objects not necessarily related to the courses list so I'm not sure why they're currently returned together.

So I didn't add pagination to `in_process_course_actions` since I believe that that list should be returned by another different view dedicated only to course reruns state objects.

What do you think? That new view is out of the scope of this PR, so this new view returns the list of course reruns objects without pagination.

Reply:
I don't think we currently need this paginated list, I don't see a problem to leave it like that for the moment, and work on it later in a dedicated PR if necessary.